### PR TITLE
docs: add community section

### PR DESCRIPTION
**Pull request by AhmedElywa**
I created tool to generate `objectType` , `queryType` and `mutationType` from `schema.prisma` file for every model.
I love to share it here in docs.
If you want to move this up or down to put is in right place in docs or change anything, you can ask me to do this.
Thanks

### CODE REVIEW

**Comment by jasonkuhrt** [comment]
Hey @AhmedElywa thanks for this. Just FYI the scope of the tool you built will be subsumed by some of our future plans. That doesn't mean we can't mention it though. Just want to be clear that once core tooling has caught up to its functionality this project would update the docs to mention it under a new "past" or "deprecated" section (or something) where caveats are made clear (this one being, there is a new idiomatic approach to solving the problem). Hope that makes sense? If you're cool with that, let's move forward with this 🙌. Let me know.

File: README.md [hunk 1]
```diff
@@ -597,6 +597,56 @@ These are tips to help you with a successful project workflow
...
+    ```prisma
+    model User {
+      id        String   @id @unique @default(cuid())
+      email     String   @unique
+      birthDate DateTime
+      posts     Post[]
+    }
+    ```
+    
+    Output
```
Comment (jasonkuhrt) [inline README.md]: Would help if you show the command / script user has to run.

File: README.md [hunk 2]
```diff
@@ -597,6 +597,56 @@ These are tips to help you with a successful project workflow
    }
    ```
 
+1. You can use [`Create nexus types`](https://github.com/AhmedElywa/create-nexus-type) tool to generate `objectType` , `queryType` and `mutationType` from your `schema.prisma` file for every model.
```
Comment (jasonkuhrt) [inline README.md]: Can you put this section under like so:

```md
### Community

#### `create-nexus-type`

...
```

**Review by jasonkuhrt — CHANGES_REQUESTED** [review]

**Comment by AhmedElywa** [comment]
Hi @jasonkuhrt  you mean that you have plane to add this function to prisma-nexus package in future and will update docs to put your tool ? I think all what i need to help people this open source project all work for help other this not a problem at all and thank you for this

File: README.md (hunk 1 above)
Comment (AhmedElywa) [reply README.md]: i will add full docs like in my Readme page if this help

File: README.md (hunk 2 above)
Comment (AhmedElywa) [reply README.md]: please can you share where i can put exactly here after any section ?
image

**Comment by jasonkuhrt** [comment]
> you mean that you have plane to add this function to prisma-nexus package in future and will update docs to put your tool ? 

We would not put that logic into this library. Something else, a higher order tool. Not announced yet.

>  I think all what i need to help people this open source project all work for help other this not a problem at all and thank you for this

IIUC this is basically 🙌 :) 

cool

File: README.md (hunk 2 above)
Comment (jasonkuhrt) [reply README.md]: A new section, after `Reference`, called `Community`, please.

Comment (AhmedElywa) [reply README.md]: Ok done

**Comment by jasonkuhrt** [comment]
@AhmedElywa Thanks! Might trim down the doc in the future, but this is a fine start. Congrats on being the first community listed project :)

**Review by jasonkuhrt — APPROVED** [review]

**Comment by AhmedElywa** [comment]
Thank you i hope i can give more to Prisma in the future i love this framework

**Pull request merged**